src/js/Content/Features/Store/App/FWaitlistDropdown.ts

```diff
@@ -156,18 +156,18 @@ export default class FWaitlistDropdown extends Feature<CApp> {
             if (parent.classList.contains("loading")) { return; }
             parent.classList.add("loading");
 
-            const result = await (new Promise<boolean>(resolve => {
-                // @ts-expect-error
-                document.addEventListener("addRemoveWishlist",
-                    (e: CustomEvent<boolean>) => resolve(e.detail)),
-                    {once: true}
-            }));
-            if (result) {
-                wishlisted = !wishlisted;
-                updateDiv();
-            } else {
-                parent.classList.remove("loading");
-            }
+            await new Promise<void>(resolve => {
+                // @ts-ignore
```

Comment:
It's better to use @ts-expect-error

I'm used to @ts-ignore too, but I need to unlearn that.
https://www.totaltypescript.com/concepts/how-to-use-ts-expect-error